### PR TITLE
FUSETOOLS-2695 - avoid NullPointerException

### PR DESCRIPTION
**Pull request by apupier**
# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [x] Did the CI job report a successful build?
- [x] Is the non-happy path working, too?

## Maintainability

- [x] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [x] Are there integration tests (or at least a jira to tackle these)?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelMavenUtils.java [hunk 1]
```diff
@@ -216,6 +216,7 @@ public Model getMavenModel(IProject project, boolean resolveFully) {
 	
 	private String getCamelVersionFromDependencies(List<Dependency> deps) {
 		for (Dependency pomDep : deps) {
+			System.out.println(pomDep);
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelMavenUtils.java]: remove sysout?

File: core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/xml/namespace/FindNamespaceHandlerSupport.java [hunk 2]
```diff
@@ -69,23 +67,20 @@ public boolean parseContents(InputSource contents) throws IOException, ParserCon
...
+		final SAXParser parser = parserFactory.newSAXParser();
+		final XMLReader reader = parser.getXMLReader();
+		//reader.setProperty("http://xml.org/sax/properties/lexical-handler", this); //$NON-NLS-1$
+		// disable DTD validation (bug 63625)
+		try {
+			//	be sure validation is "off" or the feature to ignore DTD's will not apply
+			reader.setFeature("http://xml.org/sax/features/validation", false); //$NON-NLS-1$
+			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false); //$NON-NLS-1$
+		} catch (SAXNotRecognizedException | SAXNotSupportedException e) {
+			// not a big deal if the parser does not recognize the features
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/xml/namespace/FindNamespaceHandlerSupport.java]: I am not a friend of ignoring and not logging exceptions

Comment (apupier) [reply core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/xml/namespace/FindNamespaceHandlerSupport.java]: logging now at this old code place

Comment (lhein) [reply core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/xml/namespace/FindNamespaceHandlerSupport.java]: I know its old code and not touched in this PR but I think its good to improve it.

**Pull request merged**